Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics/MicroGraphics/Driver/Graphics.MicroGraphics/Buffers/BufferRgb8888.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+
+namespace Meadow.Foundation.Graphics.Buffers
+{
+    public class BufferRgb8888 : BufferBase
+    {
+        public override int ByteCount => Width * Height * 4;
+
+        public override ColorType displayColorMode => ColorType.Format32bppRgba;
+
+        public BufferRgb8888(int width, int height, byte[] buffer) : base(width, height, buffer) { }
+
+        public BufferRgb8888(int width, int height) : base(width, height) { }
+
+        public int GetPixelInt(int x, int y)
+        {
+            //get current color
+            var index = ((y * Width) + x) * 4;
+
+            return (int)(Buffer[index] << 24 | Buffer[++index] << 16 | Buffer[++index] << 8 | Buffer[++index]);
+        }
+
+        public override Color GetPixel(int x, int y)
+        {
+            var index = ((y * Width) + x) * 4;
+
+            //split into R,G,B & invert
+            byte r = Buffer[index];
+            byte g = Buffer[index + 1];
+            byte b = Buffer[index + 2];
+            byte a = Buffer[index + 3];
+
+            return new Color(r, g, b);
```

Comment:
Alpha is lost?

Reply:
+1 we should use the constructor that takes an alpha value - new Color(r, ,g, b, a) .... great catch @pekkah  - I would have missed that :)